backend/src/app.module.ts

```diff
@@ -1,9 +1,14 @@
 import { Module } from '@nestjs/common'
-import { PrismaModule } from './prisma/prisma.module'
```

Comment:
Dlaczego to przenioslas nizej? To kwestia merge byla?

Reply:
Możliwe, że tak